### PR TITLE
[ADD] partner_contact_nutrition_diet

### PR DESCRIPTION
**Pull request by max3903**
#335 

Depends on #337

### CODE REVIEW

File: partner_contact_nutrition_diet/__init__.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Ursa Information Systems <http://ursainfosystems.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from . import models
```
Comment (lasley) [inline partner_contact_nutrition_diet/__init__.py]: newline

File: partner_contact_nutrition_diet/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Ursa Information Systems <http://ursainfosystems.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Partner Contact Nutrition Diet",
+    "summary": "Set the nutrition diet of your contacts",
+    "version": "9.0.1.0.0",
+    "category": "Uncategorized",
```
Comment (lasley) [inline partner_contact_nutrition_diet/__openerp__.py]: What do you think about putting these under `Health`? Seems like we have enough modules to warrant a category

File: partner_contact_nutrition_diet/data/nutrition_diet.xml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+     <record id="nutrition_diet_pescatarian" model="nutrition.diet">
+         <field name="name">Pescatarian</field>
+     </record>
+
+     <record id="nutrition_diet_vegetarian" model="nutrition.diet">
+         <field name="name">Vegetarian</field>
+     </record>
+
+ </odoo>
```
Comment (lasley) [inline partner_contact_nutrition_diet/data/nutrition_diet.xml]: newline

File: partner_contact_nutrition_diet/data/nutrition_diet.xml [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+     </record>
+
+     <record id="nutrition_diet_pescatarian" model="nutrition.diet">
+         <field name="name">Pescatarian</field>
+     </record>
+
+     <record id="nutrition_diet_vegetarian" model="nutrition.diet">
+         <field name="name">Vegetarian</field>
+     </record>
+
```
Comment (lasley) [inline partner_contact_nutrition_diet/data/nutrition_diet.xml:19]: Vegan?

File: partner_contact_nutrition_diet/models/__init__.py [hunk 5]
```diff
@@ -0,0 +1,8 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Ursa Information Systems <http://ursainfosystems.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from . import (
+    nutrition_diet,
+    res_partner,
+)
```
Comment (lasley) [inline partner_contact_nutrition_diet/models/__init__.py:8]: newline

File: partner_contact_nutrition_diet/views/res_partner_view.xml [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+         <field name="arch" type="xml">
+             <data>
+                 <xpath expr="//page[@name='nutrition']/group/group[@name='group2']" position="inside">
+                     <field name="nutrition_diet_id"/>
+                 </xpath>
+             </data>
+         </field>
+     </record>
+
+ </odoo>
```
Comment (lasley) [inline partner_contact_nutrition_diet/views/res_partner_view.xml]: newline

**Review by lasley — CHANGES_REQUESTED** [review]
Thanks @max3903 - comments inline

File: partner_contact_nutrition_diet/models/nutrition_diet.py [hunk 7]
```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Ursa Information Systems <http://ursainfosystems.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+from openerp import models, fields
```
Comment (susport) [inline partner_contact_nutrition_diet/models/nutrition_diet.py]: Please sort alphabetically the import according to oca guide

File: partner_contact_nutrition_diet/models/res_partner.py [hunk 8]
```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Ursa Information Systems <http://ursainfosystems.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+from openerp import models, fields
```
Comment (susport) [inline partner_contact_nutrition_diet/models/res_partner.py]: Please sort alphabetically the import according to oca guide

**Review by leemannd — APPROVED** [review]
Once Comments from @lasley are fullfilled

File: partner_contact_nutrition_diet/data/nutrition_diet.xml (hunk 4 above)
Comment (max3903) [reply partner_contact_nutrition_diet/data/nutrition_diet.xml:19]: http://www.diffen.com/difference/Vegan_vs_Vegetarian ;)

File: partner_contact_nutrition_diet/__openerp__.py (hunk 2 above)
Comment (max3903) [reply partner_contact_nutrition_diet/__openerp__.py]: ok

File: partner_contact_nutrition_diet/data/nutrition_diet.xml (hunk 4 above)
Comment (lasley) [reply partner_contact_nutrition_diet/data/nutrition_diet.xml:19]: Sorry I was suggesting that we add vegan. It's fairly common IMO

**Review by lasley — APPROVED** [review]
Thanks @max3903

**Pull request merged**